benchmarks/kane3.py

```diff
@@ -54,7 +79,17 @@
 print "Subs"
 cg = cg.subs(dict(zip(consts, range(1, len(consts)+1))))
 cg = cg.subs(dict(zip(syms, range(1, len(syms)+1))))
-print "Eval derivative"
+print "SymPy/SymEngine Eval derivative"
 print cg.n(real=True)
-print cg.n(200)
+print cg.n(200, real=True)
```

Comment:
I used complex before for a reason --- to show that the final answer is not complex. If you force it to use real, you can't do that. Though presumably you assume you would get an exception if it were to become complex?